cmd/medius/publish/docs.go

```diff
@@ -0,0 +1,67 @@
+package publish
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"kubevirt.io/containerdisks/pkg/quay"
+
+	"github.com/ghodss/yaml"
+	"kubevirt.io/containerdisks/cmd/medius/common"
+	"kubevirt.io/containerdisks/pkg/docs"
+)
+
+func NewPublishDocsCommand(options *common.Options) *cobra.Command {
+	options.PublishDocsOptions = common.PublishDocsOptions{
+		TokenFile: "",
+	}
+
+	publishCmd := &cobra.Command{
+		Use:   "docs",
+		Short: "Synchronize container disk descriptions with quay.io",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			success := true
+			client := quay.NewQuayClient(options.PublishDocsOptions.TokenFile, "containerdisks")
```

Comment:
Does not fit the current structure, but would be less redundant if the `"containerdisks"` could be extracted from `options`.

Reply:
Done.